apps/test-app/frontend/src/components/tree-widget/TreeWidget.tsx

```diff
@@ -273,3 +336,56 @@ function parseTreeNodeItem(node: HierarchyNode): DelayLoadedTreeNodeItem {
     autoExpand: node.autoExpand,
   } as DelayLoadedTreeNodeItem;
 }
+
+function createInfoNode(parentNode: TreeNodeItem | undefined, message: string, type?: InfoTreeNodeItemType): PresentationInfoTreeNodeItem {
+  const id = parentNode ? `${parentNode.id}/info-node` : `/info-node/${message}`;
```

Comment:
why aren't we including `message` if there's a `parentNode`?

Reply:
Including `message` in both cases.